app/views/CartView.php

```diff
@@ -0,0 +1,30 @@
+<?php
+include "Header.php";
+?>
+<h1>Cart</h1>
+<div style="padding: 10px; max-width: 50%; margin: auto;">
+<?php
+    if(!isset($_SESSION['cart']) || count($_SESSION['cart']['products']) <= 0)
+        echo "<p>Cart Empty!</p>";
+    else
+    {
+        foreach ($_SESSION['cart']['products'] as $item)
+        {
+            echo '<div style="background-color: burlywood; padding: 10px; margin: 10px;">Product:'
+                . $item[0]->name.' Price:' . $item[0]->price
+                . '$ Quantity:' . $_SESSION['cart'][$item[0]->id]['quantity'];
+            echo "<a class='btn btn-primary' href='cart/addQuantity?id=" . $item[0]->id . "'>+</a>";
+            echo "<a class='btn btn-primary' href='cart/removeQuantity?id=" . $item[0]->id . "'>-</a></div>";
+        }
```

Comment:
Ovako se to radi da bude preglednije i lakše za raditi. Napravi izmjene na svim template fileo-ovima.

```suggestion
        <?php foreach ($_SESSION['cart']['products'] as $item): ?>
            <div style="background-color: burlywood; padding: 10px; margin: 10px;">
                Product: <?=  $item[0]->name ?>
                Price: <?=  $item[0]->price ?>$ 
                Quantity: <?=  $_SESSION['cart'][$item[0]->id]['quantity'] ?>
                <a class="btn btn-primary" href="cart/addQuantity?id=<?= $item[0]->id ?>">+</a>
                <a class="btn btn-primary" href="cart/removeQuantity?id=<?= $item[0]->id ?>">-</a>
            </div>
        <?php endforeach; ?>
```